Makefile

```diff
@@ -30,16 +34,34 @@ checkvars:
 	@if test -z "$(TAG)"; then echo "TAG missing"; exit 1; fi
 	@if test -z "$(HUB)"; then echo "HUB missing"; exit 1; fi
 
-.PHONY: test
-test:
-	@bazel test //...
+setup:
```

Comment:
TODO: file bug to fix this, it's the only go component that needs this kind of hack (and it
is not very clear where the builders' private kube keys will end up)

Reply:
Yeah I agree it is horrible, even to build pilot you need a kube config.
https://github.com/istio/istio/issues/1511